src/main/java/ai/starlake/transpiler/JSQLColumResolver.java

```diff
@@ -266,6 +289,21 @@ public <T> T getLineage(Class<? extends TreeBuilder<T>> treeBuilderClass, String
     return builder.getConvertedTree(this);
   }
 
+  
```

Comment:
Please why do we need this method here?

Reply:
Ability to configure the underlying parser - e.g. to handle T-SQL way of quoting (square brackets)